Standards/scs-XXXX-vN-kaas_load_balancers.md

```diff
@@ -0,0 +1,126 @@
+  * So, handling `externalTrafficPolicy: Local` as "supported" may cause confusion, as client IP preservation is its most prominent feature - most likely also more prominent than the reduced number of hops
+* [Being the default setting for e.g. nginx's nginx-ingress-controller helm chart](#ootb)
+  * Ultimately, there will be always opt-in/opt-out fields in Kubernetes resources which impact cross-provider-support.
+  * The mere fact that e.g. nginx inc.'s ingress helm chart sets `externalTrafficPolicy: Local` does not mean it is some sort of "industry standard procedure". In fact, while it is not the only helm chart to set it by default, there are a few popular ingress controller charts which apparently do not: [`kubernetes/ingress-nginx` helm chart](https://github.com/kubernetes/ingress-nginx/blob/e7bee5308e84269d13b58352aeae3a6f27ea6e52/charts/ingress-nginx/values.yaml#L475), [traefik helm chart](https://github.com/traefik/traefik-helm-chart/blob/d1a2c281fb12eca2693932acbea6fec7c2212872/traefik/values.yaml), [contour helm chart](https://github.com/bitnami/charts/blob/30300ee924e6e6c55fe9069bf03791d8bcae65b7/bitnami/contour/values.yaml).
+  * Furthermore, even when `externalTrafficPolicy: Local` is required to work while not being required to preserve the client IP, this directly contradicts the nginx's helm chart parameter docs, which say that ["Local preserves the client source IP"](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/#configuration). So, even though the deployment will fundamentally work out of the box, it will work differently than it would be expected by an user who reads the nginx-ingress-controller helm chart documentation.
+* [Improve performance](#performance)
+  * Significant improvements are possible, but are not validated yet
+
+### Conclusion (**TBD**)
+
+| Option 1 | Option 2 | Option 3 |
+|----|----|----|
+| DO require SCS compliant clouds to work with `externalTrafficPolicy: Local` (in the reference implementation: enabling the health check mechanism to avoid constant connectivity problems); Do NOT require them to preserve the client IP | DO NOT require SCS compliant cloud providers to support `externalTrafficPolicy: Local` (in the reference implementation: enabling the health check mechanism to avoid constant connectivity problems if users set it anyway) | DO NOT require SCS compliant cloud providers to support `externalTrafficPolicy: Local` (Do NOT configure health checks in the reference implementation in order to not derive from upstream defaults) |
+
```

Comment:
There is another option (maybe option 0).
Require SCS compliant cloud managers to support TrafficPolicy: Local (e.g. by enabling health monitors) AND require them to preserve the original requester's IP address (e.g. by enabling some PROXY protocol setup in a way that it's transparent to the L7 nginx.

Reply:
I added a bunch of options and wanted them to "build upon" each other; This one would be number 5 right now.